control/preimage/src/main.rs

```diff
@@ -298,6 +312,58 @@ async fn run() -> Result<(), Box<dyn std::error::Error>> {
             let call2 = send_xcm_asset_hub(&context, vec![set_ethereum_fee]).await?;
             utility_force_batch(vec![call1, call2])
         }
+        Command::TreasuryProposal2024 => {
+            let beneficiary: [u8; 32] =
+                hex!("40ff75e9f6e5eea6579fd37a8296c58b0ff0f0940ea873e5d26b701163b1b325");
+
+            let mut scheduled_calls: Vec<
+                polkadot_runtime::api::runtime_types::polkadot_runtime::RuntimeCall,
+            > = vec![];
+
+            // Immediate direct payout of 191379 DOT
+            let instant_pay_amount: u128 = 1913790000000000;
+            let call = instant_payout(&context, instant_pay_amount, beneficiary).await?;
+            scheduled_calls.push(call);
+
+            // Immediate 2-year vesting payout of 323275 DOT
+            let vesting_pay_amount: u128 = 3232750000000000;
+            let vesting_period: u32 = 2 * 365 * 24 * 3600 / 6;
+            let per_block: u128 = vesting_pay_amount / (vesting_period as u128);
+            let treasury: [u8; 32] =
+                hex!("6d6f646c70792f74727372790000000000000000000000000000000000000000");
+            // consider the proposal unconfirmed, start vesting after 45 days to make sure the
+            // starting_block is valid.
+            let delay: u32 = 45 * 24 * 3600 / 6;
+            let call = vesting_payout(
+                &context,
+                vesting_pay_amount,
+                per_block,
+                treasury,
+                beneficiary,
+                delay,
+            )
+            .await?;
+            scheduled_calls.push(call);
+
+            // Scheduled payout in 75 days from now of 161637 DOT
+            let scheduled_pay_amount: u128 = 1616370000000000;
```

Comment:
Rather use compile-time arithmetic on constants to make these numbers easier to read.

```suggestion
            pub const DOT: u128 = 10_000_000_000;
            let scheduled_pay_amount: u128 = 161637 * DOT;
```

Reply:
Can depend on https://github.com/polkadot-fellows/runtimes/blob/750aa739020f0a42073d925f98f03b38d43a1888/relay/polkadot/constants/src/lib.rs#L30

Reply:
https://github.com/Snowfork/snowbridge/pull/1246/commits/603a9ef209d15d9c1123432ef5331c4daf1104e7